# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../util-linux/gcc34.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

S.th. modified is not const ... for gcc-3.4.

  - Rene Rebe <rene@exactcode.de>

--- ./mount/mount.c.vanilla	2004-07-04 20:35:50.000000000 +0200
+++ ./mount/mount.c	2004-07-04 20:36:29.000000000 +0200
@@ -1104,7 +1104,7 @@
  * Return 0 for success (either mounted sth or -a and NOAUTO was given)
  */
 static int
-mount_one (const char *spec, const char *node, char *types, const char *opts,
+mount_one (const char *spec, char *node, char *types, const char *opts,
 	   char *cmdlineopts, int freq, int pass) {
   int status;
   int status2;
